packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -88,23 +88,21 @@ export function Anthropic({
   return result;
 }
 
-interface AnthropicChatModelProps extends ModelPropsWithChildren {
-  model: ValidChatModel;
-}
-export async function* AnthropicChatModel(
-  props: AnthropicChatModelProps,
-  { render, getContext, logger }: AI.ComponentContext
-): AI.RenderableStream {
-  const messageElements = await render(props.children, {
+export async function getPolyfilledMessages(
+  children: Node,
+  render: AI.ComponentContext['render'],
+  providerName: string,
+  doesNotSupoortFunctionsErrorCode: ErrorCode
+) {
+  const messageElements = await render(children, {
     stop: (e) =>
       e.tag == SystemMessage ||
       e.tag == UserMessage ||
       e.tag == AssistantMessage ||
       e.tag == FunctionCall ||
       e.tag == FunctionResponse,
   });
-  yield AI.AppendOnlyStream;
```

Comment:
@petersalas we're changing the timing of when this is `yield`ed slightly. Does that matter?

Reply:
Rendering blocks until the first `yield`, so you generally want to `yield AI.AppendOnlyStream` as early as possible (namely before any `await`s`).

Reply:
Cool, so moving it forward in this function seems good.